### PR TITLE
Ionic+Vue Example

### PR DESCRIPTION
**Pull request by ArcticGizmo**
I have created a sample project for  Ionic+Vue3.
It is a combination of Ionic+React and the SPA Vue example.

I have tested it in the browser and on an android device, but not on iOS

### CODE REVIEW

**Comment by stevehobbsdev** [comment]
Thanks for this PR @ArcticGizmo, looks very comprehensive 🎉 

Before I do a deep dive into reviewing it, I will say that our samples typically are companions for our quickstarts that exist on our docs site ([example for Ionic React](https://auth0.com/docs/quickstart/native/ionic-react/interactive)), but we don't currently have this for Vue.

Let me have a discussion internally around timelines on that and how we want to move forward with this PR, and come back to you.

**Comment by ArcticGizmo** [comment]
@stevehobbsdev I have created a PR to add the [quickstart guide](https://github.com/auth0/docs/pull/10107) as I thought it might help out :)

**Review by frederikprijck — APPROVED** [review]
Thanks for all the work here! 🚀 

Sorry it took so long to get to this PR, but we wanted to wait for when we release v2 of Auth0-Vue.

I will merge this in a different branch, as we will need to patch a few things for v2 of our SDK, but this should land in master anytime soon.

File: .circleci/config.yml [hunk 1]
```diff
@@ -73,6 +73,13 @@ jobs:
       - build-node:
           working_directory: react
           config_path: src
+  build-vue:
+    executor:
+      name: build-executor
+    steps:
```
Comment (frederikprijck) [inline .circleci/config.yml:79]:
```suggestion
    steps:
      - attach_workspace:
          at: .
```

**Review by frederikprijck — CHANGES_REQUESTED** [review]

**Review by frederikprijck — APPROVED** [review]

**Pull request merged**